### PR TITLE
Replaced regular withdrawal-only ATM with wall mounted version on Skipper

### PR DESCRIPTION
**Pull request by sTiKyt**
## About the PR

Replaced regular ATM with a wall-mounted version to preserve some space on Skipper

Money is perfectly reachable that way

**Media**

Updated version:
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: Replaced regular ATM with a wall-mounted version to preserve some space on Skipper

### CODE REVIEW

**Comment by sTiKyt** [comment]
Basically, one of the reasons I've made #42 was to improve Skipper, and other maps that use ATMs

File: Resources/Maps/Shuttles/skipper.yml [hunk 1]
```diff
@@ -16,11 +16,22 @@ tilemap:
...
+    components:
+    - type: MetaData
+    - type: Transform
+    - type: Map
+    - type: PhysicsMap
+    - type: GridTree
+    - type: MovedGrids
+    - type: Broadphase
+    - type: OccluderTree
+    - type: LoadedMap
```
Comment (Cheackraze) [inline Resources/Maps/Shuttles/skipper.yml:29]: needs to be saved as a grid, not a map

**Review by Cheackraze — CHANGES_REQUESTED** [review]

File: Resources/Maps/Shuttles/skipper.yml (hunk 1 above)
Comment (sTiKyt) [reply Resources/Maps/Shuttles/skipper.yml:29]: Oh, i did save it as grid, but it adds those anyway

**Comment by sTiKyt** [comment]
Okay, it was acting funny for some reason, and still saved it as a map before, should be fixed now

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**